aw_transform/classify.py

```diff
@@ -7,6 +7,7 @@
 
 Tag = str
 Category = List[str]
+ProductivityScore = int
```

Comment:
Maybe have a float instead?

Reply:
Why do you think it should be a float?

Reply:
Because the world is not so black and white. Washing dishes and studying are both two productive tasks but one is more often important than the other so we need to be able to represent something in between "neutral productivity" and "productive"